### DISCUSSION TITLE
Nextjs-blog tutorial not loading on localhost:3000

### DISCUSSION THREAD

**Discussion by RaulGee** [vercel/next.js#33988] [Help]
Wanting to learn Nextjs and started by trying to go through the tutorial. Any time i 'npm run dev' local host just endlessly loads. When inspecting and checking the console I'm getting nothing at all. 
Seems like nothing is coming back from the request. 

Node version v16.13.1.

I've tried both npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter" & npm init next-app nextjs-blog.

Any help with this will be appreciated. Thank you.

**Comment by mounahaadi** [comment]
It worked for me

1. Delete the .next folder and try npm run dev command **OR**

2. Move the project to another folder location **OR**
 
3. Take back up of the code and delete the project and then run the same project again

4. You can try all of the above also

**Reply by TheArcher2002** [reply]
Why does this happen?

**Reply by vendkura** [reply]
I have deleted the .next folder, then waited for some minute  (my computer was overheating ) and rerun npm run dev. Then it worked for me.

Thank you for the tip

**Comment by w1am** [comment]
For me, the reason the dev server stopped working was that something in my cookies was causing errors. All I had to do was delete all the cookies, and it worked again.

**Reply by Erickferpinedo** [reply]
and how do you delete the cookies and wht do you mean by that?

**Comment by toygma** [comment]
Has anyone solved this problem?

**Comment by techsavvy87** [comment]
I installed Nextjs project, when run "npm run dev" on cmd. when I try to visit localhost:3000 on chrome, dev server is not working and page is not loading, "The site can't be reached" why
